draft-ietf-httpbis-priority.md

```diff
@@ -541,12 +541,19 @@ background urgency being associated. However, in the worst case, the asymmetry
 between the precedence declared by multiple clients might cause responses going
 to one end client to be delayed totally after those going to another.
 
-In order to mitigate this fairness problem, when a server responds to a request
-that is known to have come through an intermediary, the server SHOULD prioritize
-the response as if it was assigned the priority of  `u=1, i`
-(i.e. round-robin) regardless of the value of the Priority header field being
-transmitted, unless the server knows the intermediary is not
-coalescing requests from multiple clients.
+In order to mitigate this fairness problem, a server could use knowledge about
+the intermediary as another signal in its prioritization decisions. For
+instance, if a server knows the intermediary is coalescing requests, then
+it could prioritize responses using a fixed urgency level with incremental
+loading. Treating all requests from an intermediary in this way would result in
+round-robin scheduling. This works well when the network bottleneck exists
+between the intermediary and the end client, as the intermediary buffers
+responses and forwards the chunks based on the prioritization scheme it
+implements. A more sophisticated server could prioritize responses with a
+dynamic urgency that more closely reflects request urgency, in order to have
+weighted round-robin scheduling. This would result in less urgent responses
```

Comment:
I'd drop the last two sentences entirely.  Statements about sophistication will likely seem quaint in 6 months time.

Reply:
What about condensing this to say "Weighted round-robin scheduling could improve the use of contained network capacity further"?

Reply:
I think my preference goes to removing the last two sentences, due to the following reasons:
* Use of WRR degrades fairness, and might have negative impact on perceived performance of an end user.
* This is a problem that we do not need to solve now. Our choice of two priority parameters has been based on what we want to have for web browsing. For web browsing, we do not expect to see the case being described here.

Reply:
agreed, thanks!